Userland/Libraries/LibC/string.h

```diff
@@ -11,6 +11,17 @@
 
 __BEGIN_DECLS
 
+// There are a few C Standard Libraries that have completely broken the standard
+// and include <strings.h> inside <string.h>. This means that certain pieces
+// of software expect functions in <strings.h> to be available by including
+// <string.h>...
+// Considering that we'd rather be "on-spec" rather than compatible, let's
```

Comment:
Wait, who says we'd rather be "on-spec" rather than compatible? :)

Reply:
Hmm, this moreson comes from how we've followed specs when implementing things :) Want me modify this comment, or should we just include strings.h and leak the prototypes?

Reply:
Linux goes:
```
__USE_MISC       Define things from 4.3BSD or System V Unix.
```
and 
```
#if defined _DEFAULT_SOURCE
# define __USE_MISC 1
#endif
```

I personally don't see why having a few extra C function prototypes matters :thonk:

Reply:
Yeah, seems harmless to take the extra include :shrug:

Reply:
Sounds good. I'll remove the macro and comment, and update the Commit message